ocs_ci/ocs/jenkins.py

```diff
@@ -328,6 +330,15 @@ def create_ocs_jenkins_template(self):
                 "slaves.NodeProvisioner.MARGIN0=0.85",
             }
         )
+        if Version.coerce(self.ocp_version) >= Version.coerce("4.9"):
+            tmp_dict["objects"][3]["spec"]["template"]["spec"]["containers"][0][
```

Comment:
can you add some comment or include the top level yaml how it looks like?

Reply:
done